tests/datadriven/test_variants.py

```diff
@@ -84,19 +139,63 @@ def verifyVariantsEqual(self, gaVariants, pyvcfVariants):
                     self.assertEqual(str(alt1), str(alt2))
             else:
                 self.assertEqual(gaVariant.alternateBases, alt)
-            # TODO check INFO fields
 
-    def testSearchAllVariants(self):
-        allVariants = []
+            pyvcfCallMap = {}
+            for call in pyvcfVariant:
+                pyvcfCallMap[call.sample] = call
+            _verifyVariantCalls(gaVariant.calls, pyvcfCallMap, pyvcfInfo)
+            self._verifyInfoEqual(gaVariant.info, pyvcfInfo)
+
+    def _verifyGaVariantsSample(self, gaVariants, sampleIds):
+        for v in gaVariants:
```

Comment:
We should probably use `variant` here instead of `v`: a lot of people _really_ don't like single letter variables!